master/docs/manual/cfg-workers-docker.rst

```diff
@@ -285,9 +285,15 @@ In addition to the arguments available for any :ref:`Latent-Workers`, :class:`Hy
 
 .. code-block:: python
 
-    import requests, urlparse
+    try:
```

Comment:
That's what I had before @rutsky asked for the change.
OK, I've put it back to what was there before.